dev_guide/authentication.adoc

```diff
@@ -103,3 +114,20 @@ for user input to confirm (`y/N` kind of input) about connecting insecurely.
 
 CLI configuration files allow you to easily
 link:../cli_reference/manage_cli_profiles.html[manage multiple CLI profiles].
+
+// tag::systemadminlogin[]
+
+[NOTE]
+====
+If you have access to system administrator credentials but have logged out of
+the link:../architecture/core_concepts/projects_and_users.html#users[default
+system user] *system:admin*, you can log back in as this user with the following
+command as long as the credentials are still present in your *_~/.kube/config_*
```

Comment:
Thx @sdodson.

Part of why I worded it like I did was because atm the note is also being included in the `dev_guide/authentication` topic (a developer-focused topic, while the note really is addressing admins). I could drop it from `dev_guide/authentication` and just leave it in the 2 `cli_reference` topics, then simply the note to:

"If you are an administrator with access to the default system user system:admin, you can log in as this user at any time with the following command, provided your CLI configuration file retains the system:admin credentials:"

Avoids the topic of where said config file is located, for better or worse....